mlflow/R/mlflow/tests/testthat/test-model-h2o.R

```diff
@@ -49,6 +49,7 @@ test_that("can print model correctly after it is loaded", {
 })
 
 test_that("can load and predict with python pyfunct and h2o backend", {
+  h2o::h2o.shutdown(prompt = FALSE)
```

Comment:
This didn't work out as I expected.

Reply:
I thought by running `h2o.shutdown`, `h2o-python` will automatically create a new h2o cluster which should have the same versions as `h2o-python`.